src/nix/repl.cc

```diff
@@ -865,17 +876,19 @@ std::ostream & NixRepl::printValue(std::ostream & str, Value & v, unsigned int m
     return str;
 }
 
-struct CmdRepl : StoreCommand, MixEvalArgs
+struct CmdRepl : InstallablesCommand
 {
     std::vector<std::string> files;
+    Strings getDefaultFlakeAttrPaths()
+    override {
+        return {""};
+    }
+    virtual bool useDefaultInstallables() {
+        return file.has_value() or expr.has_value();
+    }
 
     CmdRepl()
     {
-        expectArgs({
-            .label = "files",
-            .handler = {&files},
-            .completer = completePath
-        });
     }
```

Comment:
Should we be removing this flag? If we do, we can delete the entire constructor; the default one will be fine.

Reply:
thx...